packages/schema/src/schema/buffer.json

```diff
@@ -7,12 +7,12 @@
   "properties": {
     "inputLayer": {
       "type": "string",
-      "description": "Select an input layer for buffering"
+      "description": "The input layer for buffering."
     },
     "bufferDistance": {
       "type": "number",
       "default": 10,
-      "description": "The distance used for buffering the geometry."
+      "description": "The distance used for buffering the geometry (degrees)."
```

Comment:
This won't always be the case. If the user selects a different `projection`, the unit will be different. 

```suggestion
      "description": "The distance used for buffering the geometry (projection units)."
```

Reply:
Thanks both, this sounds reasonable to me